sdk/eventhubs/azure_messaging_eventhubs/Cargo.toml

```diff
@@ -43,8 +43,11 @@ tokio = { workspace = true, features = ["rt-multi-thread", "macros"] }
 tracing-subscriber = { workspace = true, features = ["env-filter", "fmt"] }
 
 [features]
+default = ["native_tls"]
 test = []
 blob_checkpoint_store = []
+native_tls = ["azure_core_amqp/native_tls"]
```

Comment:
Features cannot be mutually exclusive, so is there a reason to make `native_tls` a feature? Why not just `rustls` like we did in `azure_core`?